internal/pkg/manifest/svc.go

```diff
@@ -442,3 +442,62 @@ func (s *stringOrFromCFN) UnmarshalYAML(value *yaml.Node) error {
 	}
 	return nil
 }
+
+func (cfg ImageWithPortAndHealthcheck) exposedPorts(workloadName string) (exposedPorts []ExposedPort) {
+	// Read `image.port`
+	if cfg.Port != nil {
+		exposedPorts = append(exposedPorts, ExposedPort{
+			Port:          aws.Uint16Value(cfg.Port),
+			Protocol:      "tcp",
+			ContainerName: workloadName,
+		})
+	}
+	return exposedPorts
+}
```

Comment:
https://google.github.io/styleguide/go/decisions#named-result-parameters

> Don’t use named result parameters when the names produce [unnecessary repetition](https://google.github.io/styleguide/go/decisions#repetitive-with-type).

(also, we're not really saving any amount of code with it)
```suggestion
func (cfg ImageWithPortAndHealthcheck) exposedPorts(workloadName string) () {
	// Read `image.port`
	if cfg.Port == nil {
	    return nil
	}
	return []ExposedPort{
	    {
		    Port:          aws.Uint16Value(cfg.Port),
			Protocol:      "tcp",
			ContainerName: workloadName,
	    }
	}
}
```